internal/database/database_server.go

```
@@ -0,0 +1,373 @@
+	return
+}
+
+// Start starts the database server inside a container and waits until it is ready to accept connections.
+func (s *Server) Start(ctx context.Context) error {
+	// Generate a random password for the database administrator:
+	password := uuid.NewString()
+
+	// Start the database server:
+	runOut := &bytes.Buffer{}
+	runErr := &bytes.Buffer{}
+	runCmd := exec.Command(
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)